scripts/prepack.ts

```diff
@@ -14,7 +14,7 @@ const NPM_BUILD_DIR = 'build/npm';
 const BUILD_DIR = 'build';
 
 const ASSETS = ['README.md', 'LICENSE', 'package.json', '.npmignore'];
-const ENTRY_POINTS = ['main', 'module', 'types'];
+const ENTRY_POINTS = ['main', 'module', 'types', 'browser'];
 
 const packageWithBundles = !process.argv.includes('-noBundles');
```

Comment:
Two questions here:

1) Can we change this to using two dashes, since that's the standard for full-word CLI options?
2) There are more packages without bundles than with bundles. What do you think of switching it to `--includeBundles` or something, and just adding it for those packages?

Reply:
UPDATE: Chatted IRL and I'm going to make these changes.